client/knowledge_base_search/src/containers/SearchContainer/styles.scss

```diff
@@ -139,6 +139,9 @@
     color: color('white', 10);
     background-image: url(triangle(color('white', 10), $size: 1.5));
     padding-right: 1.7rem;
+    width: auto;
+    margin-bottom: 1rem;
+    margin-top: -1rem;
```

Comment:
remove this negative margin and fix the margin on the `.feature_select--list` to `1rem` instead of `2rem` on the bottom